src/components/TitleItem.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+  <h1>
+    {{ props.title }}
```

Comment:
As I said, props are exposed automatically, so you can access them straight away.
```vue
<h1>{{ title }}</h1>
```